gcp.go

```diff
@@ -62,14 +64,23 @@ func copyDirectory(bucket storage.BucketHandle, localPath string) (int, error, [
 	objectNum := 0
 
 	// ローカルのディレクトリ構造を読み込み
-	files, err := os.ReadDir(localPath)
+	filePaths := []string{}
+	err := filepath.Walk(localPath, func(path string, info fs.FileInfo, err error) error {
+		if err != nil {
+			return err
+		}
+		if !info.IsDir() {
+			filePaths = append(filePaths, path)
+		}
+		return nil
+	})
 	if err != nil {
-		return 0, err, errs
+		return 0, err, nil
 	}
 
 	// 指定のディレクトリのファイルを1つずつストレージにコピー
-	for _, file := range files {
-		err = copyFile(bucket, localPath+"/"+file.Name(), file.Name())
+	for _, filePath := range filePaths {
+		err = copyFile(bucket, localPath+"/"+filePath, filePath)
```

Comment:
```suggestion
		err = copyFile(bucket, filePath, filePath)
```
`filePath`には`localPath`の部分も含まれているのでいらないみたいでした 🖖

Reply:
なるほどそれは知らなかった
オブジェクト名からのlocalPath部分の削除はいらないですかね？ちょっとめんどそうっちゃめんどそうですけど